<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TipsDlg</class>
 <widget class="QDialog" name="TipsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TipsDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#wgt_main{
background:#262626;
border:1px solid #585858;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wgt_main" native="true">
     <property name="styleSheet">
      <string notr="true">#wgt_top{
background:#2F2F2F;
border:none;
}
#lbl_content{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#DCDCDC;
background:transparent;
padding-left:30px;
}
#btn_confirm{
background:transparent;
background-image: url(:/TQLive/images/btn_confirm.png);
background-repeat:no-repeat;
border:none;
}
#btn_confirm:hover{
background-image: url(:/TQLive/images/btn_confirm_hover.png);
}
#btn_confirm:pressed{
margin:1px;
padding:1px;
}
#btn_cancel{
background:transparent;
background-image: url(:/TQLive/images/btn_cancel.png);
background-repeat:no-repeat;
border:none;
}
#btn_cancel:hover{
background-image: url(:/TQLive/images/btn_cancel_hover.png);
}
#btn_cancel:pressed{
margin:1px;
padding:1px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QFrame" name="wgt_top">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>34</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#btn_close{
background:transparent;
background-image: url(:/TQLive/images/pb_close_pressed.png);
background-repeat:no-repeat;
border:none;
}
#btn_close:hover{
background-image: url(:/TQLive/images/pb_close_normal.png);
}
#btn_close:pressed{
margin:1px;
padding:1px;
}
QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#4AB134;
background:transparent;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>19</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_tip">
           <property name="text">
            <string>提示</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_close">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>13</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>13</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_content">
        <property name="text">
         <string>确认解禁操作吗？</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_confirm">
          <property name="minimumSize">
           <size>
            <width>82</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_cancel">
          <property name="minimumSize">
           <size>
            <width>82</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TQLive.qrc"/>
 </resources>
 <connections/>
</ui>
